clearing-house/src/tests/open_position.rs

```diff
@@ -555,42 +555,45 @@ proptest! {
 
             VammPallet::set_price(Some(new_price));
             // Reduce (close) position by desired percentage
-            let base_amount_to_close = fraction.saturating_mul_int(base_amount);
-            let base_value_to_close = new_price.saturating_mul_int(base_amount_to_close);
-            assert_ok!(
-                <TestPallet as ClearingHouse>::open_position(
-                    &ALICE,
-                    &market_id,
-                    direction.opposite(),
-                    base_value_to_close,
-                    base_amount_to_close,
-                ),
+            let base_amount_to_close = fraction.mul_floor(base_amount);
+            let base_value_to_close = new_price
+                .try_mul(&base_amount_to_close.into_decimal().unwrap())
+                .unwrap()
+                .into_balance()
+                .unwrap();
+            let swapped = <TestPallet as ClearingHouse>::open_position(
+                &ALICE,
+                &market_id,
+                direction.opposite(),
+                dbg!(base_value_to_close),
```

Comment:
[Note to myself] 
Was that `dbg` removed in the later commits?

Reply:
Yeah, it seems it's still in the latest code this `dbg` call.
Should we keep it? 

cc/ @0xangelo

Reply:
Good catch, I'll remove it next time.